larray_editor/editor.py

```diff
@@ -817,10 +750,10 @@ def save_script(self):
         group_box_layout = QGridLayout()
         # overwrite
         radio_button_overwrite = QRadioButton("Overwrite file")
-        radio_button_overwrite.setChecked(True)
         group_box_layout.addWidget(radio_button_overwrite, 0, 0)
         # append to
         radio_button_append = QRadioButton("Append to file")
+        radio_button_append.setChecked(True)
```

Comment:
this change of the default value seems strange to me. Isn't overwrite less surprising/more usual?

Reply:
It depends how you plan to use the feature "Save Command History ...". Personally, I use it to write functions of models step by step. I try different things until I get something satisfactory and then I save the last line. Then I move to the next step.

Reply:
I am not saying it's not useful. It is indeed probably more useful this way, but I think overwrite would be less surprising. In most programs when you save as something, it overwrites the file by default. I am fine either way, we can change it later anyway.